styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -1000,6 +1045,31 @@ public Subparser parser(Subparsers subCommands, CliContext cliContext) {
     }
   }
 
+  private enum AuthCommand {
+    TEST("", "Test authorization");
+
+    private final String alias;
+    private final String description;
+
+    AuthCommand(String alias, String description) {
+      this.alias = alias;
+      this.description = description;
+    }
+
+    public Subparser parser(Subparsers subCommands, CliContext cliContext) {
```

Comment:
This is duplicated everywhere. :( Not part of this PR, but we can consider refactoring this as a default method into an interface somehow.